### PR TITLE
Fonts

### PR DESCRIPTION
**Pull request by diegomura**
- Add `fontFamily` style to some fonts that didn't have
- Refactor Article template
- Remove `d3-array` dependency

### CODE REVIEW

File: src/components/Document.js [hunk 1]
```diff
@@ -17,20 +15,6 @@ const styles = StyleSheet.create({
 })
 
 const logReactTree = (node, depth = 0) => {
```
Comment (tpreusse) [inline src/components/Document.js]: you can drop the whole function if you want

File: src/lib/fonts.js [hunk 2]
```diff
@@ -1,10 +1,6 @@
-import {
-  Font
-} from '@react-pdf/core'
+import { Font } from '@react-pdf/core'
 
-if (process.env.NODE_ENV !== 'production') {
-  require('dotenv').config()
-}
+require('dotenv').config()
```
Comment (tpreusse) [inline src/lib/fonts.js]: I'd prefer follow [code snippet](https://github.com/orbiting/proxy/blob/47779db8b146bdcc89d39acf0b54ffa7c59f7f2e/index.js#L8):
```
const DEV = process.env.NODE_ENV && process.env.NODE_ENV !== 'production'
if (DEV || process.env.DOTENV) {
  require('dotenv').config()
}
```

This allows to use the app without installing dev dependencies. And if for example `dotenv` became malicious and started stealing credentials it wouldn't run in prd. To test production locally with a `.env` file you can than use `DOTENV=1 NODE_ENV=production npm start`.

File: src/components/H2.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ const styles = StyleSheet.create({
   subheadline: {
     fontSize: 22,
     marginBottom: 5,
-    fontFamily: fontFamilies.serifBold
+    fontFamily: fontFamilies.serifTitle
```
Comment (tpreusse) [inline src/components/H2.js]: the naming here might be a bit confusing but `serifBold` is the correct one for subheadlines. `serifTitle` is more of a logo font exclusively used for the (main/first) headline of an article.

File: src/components/Infobox.js [hunk 4]
```diff
@@ -14,7 +15,8 @@ const styles = StyleSheet.create({
     paddingTop: 5,
     marginBottom: 5,
     borderTopWidth: 1,
-    borderTopColor: 'black'
+    borderTopColor: 'black',
+    fontFamily: fontFamilies.serifTitle
```
Comment (tpreusse) [inline src/components/Infobox.js]: should be `fontFamilies.serifBold`

File: src/components/Legend.js [hunk 5]
```diff
@@ -8,7 +8,7 @@ const styles = StyleSheet.create({
     marginBottom: 10,
     width: 500,
     fontSize: 12,
-    fontFamily: fontFamilies.sansSerifRegular
+    fontFamily: fontFamilies.serifRegular
```
Comment (tpreusse) [inline src/components/Legend.js]: `sansSerifRegular` is the correct one for image legends

File: src/lib/fonts.js (hunk 2 above)
Comment (diegomura) [reply src/lib/fonts.js]: Agreed!

**Comment by diegomura** [comment]
Thanks for those corrections!

**Pull request merged**